.github/workflows/automated-release-tasks.yml

```diff
@@ -0,0 +1,63 @@
+name: automated-release-tasks
```

Comment:
Job copied from `dev-4` job of the same name. Needed so that `dev-5` (soon to be `dev`) can take over automatic release branch creation once it becomes Default.